examples/end-to-end-examples/support_chatbot/scripts/convert_composer_to_hf_transformers.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2022 MosaicML Examples authors
```

Comment:
I don't think you should need this script for anything? Am I wrong?

Reply:
was under the assumption that it would be needed ~~ checkpoints saved from finetuning are .pt files but deployment takes hf checkpoints